src/backend/renderer/multigpu/mod.rs

```diff
@@ -1449,7 +1449,7 @@ impl MultiTexture {
         })))
     }
 
-    fn get<A: GraphicsApi + 'static>(
+    pub fn get<A: GraphicsApi + 'static>(
```

Comment:
Could you add a doc-comment for this function please?

```suggestion
    /// Retrieves the internal texture for a given render node, if any.
    pub fn get<A: GraphicsApi + 'static>(
```

Reply:
i just saw that you wrote a suggestion for the doc, i wrote something looking at other public get methods, please let me know if it is ok or you prefer the shorter version

Reply:
Nah seems good!